### PR TITLE
PARQUET-2005: Upgrade thrift to 0.14.1

### PR DESCRIPTION
**Pull request by gszadovszky**
Make sure you have checked _all_ steps below.

### Jira

- [ ] My PR addresses the following [Parquet Jira](https://issues.apache.org/jira/browse/PARQUET/) issues and references them in the PR title. For example, "PARQUET-1234: My Parquet PR"
  - https://issues.apache.org/jira/browse/PARQUET-XXX
  - In case you are adding a dependency, check if the license complies with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).

### Tests

- [ ] My PR adds the following unit tests __OR__ does not need testing for this extremely good reason:

### Commits

- [ ] My commits all reference Jira issues in their subject lines. In addition, my commits follow the guidelines from "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)":
  1. Subject is separated from body by a blank line
  1. Subject is limited to 50 characters (not including Jira issue reference)
  1. Subject does not end with a period
  1. Subject uses the imperative mood ("add", not "adding")
  1. Body wraps at 72 characters
  1. Body explains "what" and "why", not "how"

### Documentation

- [ ] In case of new functionality, my PR adds documentation that describes how to use it.
  - All the public functions and the classes in the PR contain Javadoc that explain what it does

### CODE REVIEW

**Review by Fokko — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -35,19 +35,19 @@ Parquet-MR uses Maven to build and depends on the thrift compiler (protoc is now
...
 chmod +x ./configure
 ./configure --disable-libs
 sudo make install
 ```
 
-If you're on OSX and use homebrew, you can instead install Thrift 0.13.0 with `brew` and ensure that it comes first in your `PATH`.
+If you're on OSX and use homebrew, you can instead install Thrift 0.14.1 with `brew` and ensure that it comes first in your `PATH`.
 
 ```
 brew install thrift@0.13
```
Comment (Fokko) [inline README.md:49]:
```suggestion
brew install thrift
```

Just. checked, and 0.14.1 is the latest version:
```
fokkodriesprong@MacBook-Pro parquet-mr % brew install thrift
Warning: thrift 0.14.1 is already installed and up-to-date.
To reinstall 0.14.1, run:
  brew reinstall thrift
```

**Comment by Fokko** [comment]
Thanks @gszadovszky

**Comment by dongjoon-hyun** [comment]
It's great! Thanks.

**Pull request merged**